src/lib/skilling/skills/magic/castables.ts

```diff
@@ -42,5 +62,261 @@ export const Castables: Castable[] = [
+		xp: 83,
+		level: 80,
+		ticks: 3,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Sapphire amulet'),
+		name: 'String Sapphire',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Water rune', 5)
+			.add('Earth rune', 10)
+			.add('Sapphire amulet (u)', 1),
+		output: new Bank().add('Sapphire amulet', 1),
+		xp: 83,
+		level: 80,
+		ticks: 3,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Emerald amulet'),
+		name: 'String Emerald',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Water rune', 5)
+			.add('Earth rune', 10)
+			.add('Emerald amulet (u)', 1),
+		output: new Bank().add('Emerald amulet', 1),
+		xp: 83,
+		level: 80,
+		ticks: 3,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Ruby amulet'),
+		name: 'String Ruby',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Water rune', 5)
+			.add('Earth rune', 10)
+			.add('Ruby amulet (u)', 1),
+		output: new Bank().add('Ruby amulet', 1),
+		xp: 83,
+		level: 80,
+		ticks: 3,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Diamond amulet'),
+		name: 'String Diamond',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Water rune', 5)
+			.add('Earth rune', 10)
+			.add('Diamond amulet (u)', 1),
+		output: new Bank().add('Diamond amulet', 1),
+		xp: 83,
+		level: 80,
+		ticks: 3,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Dragonstone amulet'),
+		name: 'String Dragonstone',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Water rune', 5)
+			.add('Earth rune', 10)
+			.add('Dragonstone amulet (u)', 1),
+		output: new Bank().add('Dragonstone amulet', 1),
+		xp: 83,
+		level: 80,
+		ticks: 3,
+		qpRequired: 50
+	},
+	{
+		id: itemID('Plank'),
+		name: 'Plank Make Logs',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Nature rune', 1)
+			.add('Earth rune', 15)
+			.add('Logs', 1),
+		output: new Bank().add('Plank', 1),
+		xp: 88,
+		level: 90,
+		ticks: 3,
+		qpRequired: 50,
+		gpCost: 70
+	},
+	{
+		id: itemID('Oak plank'),
+		name: 'Plank Make Oak',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Nature rune', 1)
+			.add('Earth rune', 15)
+			.add('Oak logs', 1),
+		output: new Bank().add('Oak Plank', 1),
+		xp: 88,
+		level: 90,
+		ticks: 3,
+		qpRequired: 50,
+		gpCost: 175
+	},
+	{
+		id: itemID('Teak plank'),
+		name: 'Plank Make Teak',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Nature rune', 1)
+			.add('Earth rune', 15)
+			.add('Teak logs', 1),
+		output: new Bank().add('Teak plank', 1),
+		xp: 88,
+		level: 90,
+		ticks: 3,
+		qpRequired: 50,
+		gpCost: 350
+	},
+	{
+		id: itemID('Mahogany plank'),
+		name: 'Plank Make Mahogany',
+		input: new Bank()
+			.add('Astral rune', 2)
+			.add('Nature rune', 1)
+			.add('Earth rune', 15)
+			.add('Mahogany logs', 1),
+		output: new Bank().add('Mahogany plank', 1),
+		xp: 88,
+		level: 90,
+		ticks: 3,
+		qpRequired: 50,
+		gpCost: 1050
+	},
+	{
+		id: itemID('Amulet of glory (4)'),
+		name: 'Recharge Glory',
+		input: new Bank()
+			.add('Astral rune', 1)
+			.add('Water rune', 5)
```

Comment:
this consumes 4 water runes in game not 5, 
and the xp is 97.5 per item and require 89 magic.

and you cast it for a whole inventory and not one item.

same goes for all other Recharge Dragonstone below.